recipes/xrftomo/meta.yaml

```diff
@@ -0,0 +1,60 @@
+
+{% set name = "XRFtomo" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }} 
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomography/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 986c4ea85c7fe8bcef274b7085209b74e8a93acd444ad1f9b947fb3f1352cbbb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - dxchange
+    - dxfile
+    - h5py
+    - matplotlib
+    - numpy
+    - pandas
+    - pyqt >=5
+    - pyqtgraph
+    - python >=3
+    - scipy
+    - seaborn
+    - setuptools
+    - scikit-image
+    - tomopy
+
+test:
+  imports:
+    - xrftomo
+
+about:
+  home: https://github.com/tomography/XRFtomo
+  license: BSD-3
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Tomography software for APS beamlines'
+
+  description: |
+    XRFtomo brings together a variety of image processing tools used for preparing, 
+    processing and reconstructing 3D renditions of samples imaged using a 
+    synchrotron-based x-ray fluorescence (XRF) microprobe or nanoprobe at the 
+    Advanced Photon Source (APS).
+
+  doc_url: https://xrftomo.readthedocs.io/en/latest/
```

Comment:
This page doesn't exist.

Reply:
My mistake, it was removed temporarily until documentation can be completed offline first. I will remove this line for now.